spec/models/manageiq/providers/vmware/infra_manager/refresher_spec.rb

```diff
@@ -335,6 +338,16 @@ def assert_specific_storage_cluster
     expect(@storage_cluster.children).to include(@child_storage)
   end
 
+  def assert_specific_storage_profile
+    @storage_profile = StorageProfile.find_by(:name => "Virtual SAN Default Storage Policy")
+    expect(@storage_profile).to have_attributes(
+      :ems_id       => @ems.id,
+      :ems_ref      => "aa6d5a82-1c88-45da-85d3-3d74b91a5bad",
+      :name         => "Virtual SAN Default Storage Policy",
+      :profile_type => "REQUIREMENT"
+    )
+  end
+
   def assert_specific_host
```

Comment:
Suggest to add some tests for cases:
* a storage will find its attached profile(s)
* a profile will find its associated storages
* Detaching/Deleting a profile will see the association gone